packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -0,0 +1,131 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// external resources
+import { expect } from 'chai';
+import { ethers } from 'ethers';
+import { AliasAccount } from '../clients/servicesClient';
+import Assertions from '../helpers/assertions';
+import { ContractFunctionParameters } from '@hashgraph/sdk';
+
+// local resources
+import parentContractJson from '../contracts/Parent.json';
+import { predefined } from '../../../relay/src/lib/errors/JsonRpcError';
+
+describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
```

Comment:
Acceptance test is good put i wanted an `hbarLimiter.spec.ts` under the `packages/relay/tests/lib` dir.
In this case I wanted to see simple unit tests where you tried out `shouldLimit`, `addExpense` and `shouldResetLimiter` with different variations for full coverage.

Given you have acceptance support you can pul this out into a separate PR so we're not holding this PR up

Reply:
I've opened a new issue to cover this https://github.com/hashgraph/hedera-json-rpc-relay/issues/582